tests/backup/backup_restore_basic_test.go

```diff
@@ -983,6 +983,8 @@ var _ = Describe("{AllNSBackupWithIncludeNewNSOption}", func() {
 			log.FailOnError(err, "Fetching px-central-admin ctx")
 			err = CreateSourceAndDestClusters(orgID, "", "", ctx)
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of source [%s] and destination [%s] clusters with px-central-admin ctx", SourceClusterName, destinationClusterName))
+			err = SetDestinationKubeConfig()
+			log.FailOnError(err, "Switching context to destination cluster failed")
```

Comment:
Move this to the step where you are scheduling the applications

```
Schedule applications to create new namespaces
```

Reply:
Done.